pkg/registry/tar.go

```diff
@@ -0,0 +1,66 @@
+package registry
+
+import (
+	"archive/tar"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"time"
+)
+
+// fsToTarGZ writes the filesystem represented by fsys to w as a gzipped tar archive.
```

Comment:
Is this still accurate?  Method name mismatch, and I didn't see any compression.